packages/tasks/src/model-libraries.ts

```diff
@@ -116,6 +116,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path_extension:"pth"`,
 		snippets: snippets.audioseal,
 	},
+	bagel: {
+		prettyLabel: "Bagel",
+		repoName: "Bagel",
+		repoUrl: "https://github.com/ByteDance-Seed/Bagel/",
+		filter: false,
+		countDownloads: `path:"llm_config.json"`,
```

Comment:
Since you already have a `config.json` we can just default to that? https://huggingface.co/ByteDance-Seed/BAGEL-7B-MoT/blob/main/config.json (I'm assuming you download this when you run inference on the model)
```suggestion

```

Reply:
ah okay! then it;s okay as is!